src/main/java/cloud/fogbow/ras/core/InteroperabilityPluginInstantiator.java

```diff
@@ -9,83 +9,93 @@
 import cloud.fogbow.ras.core.plugins.mapper.FederationToLocalMapperPlugin;
 
 import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Properties;
 
 public class InteroperabilityPluginInstantiator {
-    private Properties properties;
-    private String cloudConfPath;
-    private String mapperConfPath;
     private PluginFactory pluginFactory;
+    private Map<String, Properties> cloudPropertiesCache;
 
     public InteroperabilityPluginInstantiator(String cloudName) {
```

Comment:
We don't need a parameter for the constructor.

Reply:
Nice catch, forgot to remove it.